x/bank/client/rest/grpc_query_test.go

```diff
@@ -0,0 +1,71 @@
+package rest_test
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	grpctypes "github.com/cosmos/cosmos-sdk/types/grpc"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+	"github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+func (s *IntegrationTestSuite) TestTotalSupplyGRPCHandler() {
+	val := s.network.Validators[0]
+	baseURL := val.APIAddress
+
+	testCases := []struct {
```

Comment:
I would like to see 2 other test cases:

1. do query paramters work? The HTTP endpoint below takes a `?status=` query param.

https://github.com/cosmos/cosmos-sdk/blob/f87624e9deace3b1ca15e68110820ad76ad6c9b0/proto/cosmos/staking/v1beta1/query.proto#L15

2. Putting `?height=` is confusing here. Maybe we should add a test with `?height=3` and `grpctypes.GRPCBlockHeightHeader: "2",` and check that the "2" takes precedence.